xdcs-frontend/src/app/view/task-creation/task-creation.component.html

```diff
@@ -68,6 +68,41 @@ <h2>Resources</h2>
         <th scope="row">Kernel name</th>
         <td>{{deployment.config.kernelname}}</td>
       </tr>
+      <tr>
+        <th scope="row">Global work shape</th>
+        <td>
+          <div class="shape-dimensions">
+            <div class="shape-dimension" *ngFor="let i of [0,1,2]">
+                  <input class="form-control"
+                         type="number"
+                         min="1"
+                         required="required"
+                         [(ngModel)]="globalWorkShape[i]"
+                         placeholder="Dimension size"/>
+            </div>
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row">Local work shape</th>
+        <td>
+          <label>
+            <input type="checkbox" name="customLocalWorkShape"  [(ngModel)]="customLocalWorkShape">
+            Specify custom local work shape
+          </label>
+          <div class="shape-dimensions">
+            <div class="shape-dimension" *ngFor="let i of [0,1,2]">
+                <input class="form-control"
+                       type="number"
+                       min="1"
+                       required="required"
+                       [disabled]="!customLocalWorkShape"
```

Comment:
jak ktoś wpisze wartości i potem odznaczy checkboxa to co się wyśle?

Reply:
wyśle się null, jest to zabezpieczone